learning/basics/task2/PW-if_statement.py

```diff
@@ -0,0 +1,37 @@
+name = input("Hi there. What's your name? ")
+
+if name == "":
+
+    print("Give me at least one letter")
+
+elif not name.isalpha():
+    print("You've added at least one digit to your name (by accident I hope)")
+
+else:
+    print(f"Nice to meet you {name}")
+
+    maths_op = str(input("Wanna multiply two numbers or add them? If multiply type M, If addition type A: "))
```

Comment:
In this state if user provides wrong name your code just prints message and end process. Consider to add possibility for user to type name once more, or as long as he/she provide correct name. You can use `while` loop for that purpose:
```suggestion
name == ""
while name == "" or not name.isalpha():
    if name == "":
        name = input("Give me at least one letter")

    elif not name.isalpha():
        name = input("You've added at least one digit to your name (by accident I hope)")


print(f"Nice to meet you {name}")

maths_op = str(input("Wanna multiply two numbers or add them? If multiply type M, If addition type A: "))
```